bindata/assets/deployments/console-deployment.yaml

```diff
@@ -70,11 +70,15 @@ spec:
               path: /health
               port: 8443
               scheme: HTTPS
-            initialDelaySeconds: 150
```

Comment:
Glad to see this changed since the startup probe means that as soon as the server is healthy it will properly handled.